test/ir_Fujitsu_test.cpp

```diff
@@ -0,0 +1,149 @@
+  EXPECT_EQ(FUJITSU_AC_SWING_BOTH, fujitsuACSender.getSwing());
+  EXPECT_EQ(FUJITSU_AC_CMD_STAY_ON, fujitsuACSender.getCmd());
+
+  irsend.reset();
+  irsend.sendFujitsuAC(fujitsuACSender.getRaw(), 16);
+  EXPECT_EQ(
+  "m3224s1574m448s367m448s367m448s1182m448s367m448s1182m448s367m448s367m448"
+  "s367m448s1182m448s1182m448s367m448s367m448s367m448s1182m448s1182m448s367"
+  "m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s367m448s1182"
+  "m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s367"
+  "m448s367m448s1182m448s367m448s367m448s367m448s367m448s367m448s367m448s367"
+  "m448s367m448s1182m448s1182m448s367m448s367m448s367m448s367m448s367m448s367"
+  "m448s367m448s367m448s367m448s1182m448s1182m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s1182m448"
+  "s1182m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448"
+  "s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s8100",
+  irsend.outputStr());
+}
+TEST(TestSendFujitsuAC, GenerateShortMessage) {
+  IRFujitsuAC fujitsuACSender = IRFujitsuAC(4);
+  IRsendTest irsend(4);
+  fujitsuACSender.begin();
+  irsend.begin();
+
+  fujitsuACSender.off();
+
+  EXPECT_EQ(FUJITSU_AC_CMD_TURN_OFF, fujitsuACSender.getCmd());
+
+  irsend.reset();
+  irsend.sendFujitsuAC(fujitsuACSender.getRaw(), 8);
+  EXPECT_EQ(
+  "m3224s1574m448s367m448s367m448s1182m448s367m448s1182m448s367m448s367m448"
+  "s367m448s1182m448s1182m448s367m448s367m448s367m448s1182m448s1182m448s367"
+  "m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s367m448s1182"
+  "m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s367"TODO
+  "m448s367m448s1182m448s367m448s367m448s367m448s367m448s367m448s367m448s367"
+  "m448s367m448s1182m448s1182m448s367m448s367m448s367m448s367m448s367m448s367"
+  "m448s367m448s367m448s367m448s1182m448s1182m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448s1182m448"
+  "s1182m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448s367m448"
+  "s367m448s367m448s367m448s367m448s367m448s1182m448s367m448s367m448s367m448"
+  "s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s1182m448s8100",
+  irsend.outputStr());
+}
```

Comment:
And there's a whopping TODO in the middle so it doesn't even compile because I didn't work out the expected result yet... Didn't mean to commit that yet. Anyway, all fixed and committed now